### ISSUE TITLE
[Bug] `/` can't focus the search bar when the page finished loading

### ISSUE THREAD

**Issue by wenfangdu** [vuejs/docs#2240]
My `cmd+k` is already in use, so I use `/` to focus the search bar. However, `/` doesn't work when the page has just finished loading, it works only if I have opened the search widget before.

**Comment by viniciusteixeiradias** [comment]
@wenfangdu https://github.com/vuepress/vuepress-next/pull/1323

**Comment by Mister-Hope** [comment]
Vuejs official docs use VitePress, so wrong repo.

**Comment by viniciusteixeiradias** [comment]
@Mister-Hope Yes, my bad. Now it is right https://github.com/vuejs/vitepress/pull/2328.

**Comment by brc-dd** [comment]
Still wrong repo. It does not use the default VitePress theme. It uses https://github.com/vuejs/theme. But yeah we could consider it at the default theme too, so keeping that open for now.

**Comment by viniciusteixeiradias** [comment]
@brc-dd Omg, sorry hahaha. I will check the repo https://github.com/vuejs/theme.
